src/batch-client/types.ts

```diff
@@ -50,9 +50,23 @@ export type UserAgent = {
 	version: string;
 };
 
+export type writeSessionId = (sessionId: string) => void;
+
+export type readSessionId = () => string;
+
+export type Session = {
```

Comment:
we already have this as part of BaseRequestOptions#sessionId

Reply:
updated